File: Linguaggio_Alto_Livello/Python/Hard/word_frequency.py
text = "Python rende la programmazione piu' accessibile"
words = [word.strip(".,'\"") for word in text.lower().split()]
frequencies = {}
for word in words:
    frequencies[word] = frequencies.get(word, 0) + 1
print("Frequenze:")
for word, count in frequencies.items():
    print(f"{word}: {count}")

File: Linguaggio_Alto_Livello/JavaScript/Hard/word_frequency.js
const text = "JavaScript rende il web dinamico";
const words = text
  .toLowerCase()
  .replace(/[.,]/g, "")
  .split(/\s+/);

const frequencies = {};
for (const word of words) {
  frequencies[word] = (frequencies[word] ?? 0) + 1;
}

console.log("Frequenze:");
for (const [word, count] of Object.entries(frequencies)) {
  console.log(`${word}: ${count}`);
}
